README.md

```diff
@@ -46,6 +46,32 @@ any of their subsidiaries:
 - "Target Corporation" (for union busting and anti-union propaganda)
 - "The H&M group" (for abusive treatment of workers)
 - "Tesla, Inc" (for abusive treatment of workers)
+- "Boston Public Schools" (for working with ICE)
+- "Hewlett Packard Enterprise" (for working with ICE)
+- "Salesforce" (for working with CBP)
+- "University of Maryland" (for working with ICE)
+- "Virginia Tech" (for working with ICE)
+- "University of Alabama Birmingham" (for working with ICE)
+- "General Dynamics Corp" (for sketchy work with DHH)
```

Comment:
Can you clarify `sketchy` - feels opinionated.

Reply:
I would also suspect that DHH should be corrected to DHS, if that's what was meant.